app/controllers/members_controller.rb

```diff
@@ -62,6 +84,10 @@ def graph
     @link_data_str = edges.to_json.html_safe
   end
 
+  def check_email_confirmed?
```

Comment:
That's not the convention, when you want to say `member.present?` you use methods with question marks, so `check_email_confirmed`.